.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
-  - 5.5
```

Comment:
why removing testing on 5.5 (while still officially supporting it in composer.json) if the library supports both guzzle 6 and 7 (and so stays compatible with PHP 5.5) ?

Reply:
It is no longer part of the default image. As far as i found out the new image is now `xenial` and not longer `trusty`. 
We could use `dist: trusty` but i have read that this will not work with php 7+ on the other hand but that may be wrong.

Reply:
Alright that should be resolved now, trusty also does support 7.4+.
I did drop hhvm support as it will always fail since: https://github.com/composer/composer/issues/8127#issuecomment-489513083